netty/src/main/java/io/grpc/netty/InternalProtocolNegotiators.java

```diff
@@ -31,6 +33,36 @@ public final class InternalProtocolNegotiators {
 
   private InternalProtocolNegotiators() {}
 
```

Comment:
@ejona86, Is it ok to add this new API? I think we need this to access https://github.com/grpc/grpc-java/blob/9b0c19e698f0aa28b10f1c6513b5fb51b3e21954/netty/src/main/java/io/grpc/netty/ProtocolNegotiators.java#L685. Please let me if there's a way to do this without making changes to this file

Reply:
Yeah, adding this API is okay. We'd expect s2a to need to dig deep into Netty at times. The benefit of having it in the same repository is we can use internal APIs easily.